### PR TITLE
A brief introduction of 3SF

### PR DESCRIPTION
**Pull request by banhday**

### CODE REVIEW

File: reports/intro.tex [hunk 1]
```diff
@@ -2,15 +2,49 @@
...
+Maximal Extractable Value (MEV) exploits, 
+which could undermine the network’s integrity.
+Additionally, the extended finalization period forces users to weigh the
+trade-off between economic security and transaction speed.
+
+To address these issues and speed up finality, D’Amato et al.~\cite{d20243} have recently introduced the \emph{3-slot-finality} (3SF) protocols 
+for Ethereum that achieve finality within three slots after a proposal, hence realizing 3-slot finality.
+This feature is particularly beneficial in practical scenarios where periods of synchrony and robust honest participation often 
+last much longer than the time needed for finalization in the 3SF protocol.
+Finally, the 3SF protocol enhances the practicality of large-scale blockchain networks by enabling the dynamically-available component
```
Comment (saltiniroberto) [inline reports/intro.tex:22]: we have yet to define what the dynamically-available component is

File: reports/intro.tex [hunk 2]
```diff
@@ -2,15 +2,49 @@
...
+This design is based on the \emph{ebb-and-flow} approach introduced in~\cite{neu2021ebb}. 
+An ebb-and-flow protocol comprises two sub-protocols, each with its own confirmation rule, and each outputting a chain, with one serving 
+as a prefix of the other. 
+The first confirmation rule defines what is known as the \emph{available chain}, which provides liveness under dynamic participation
+(and synchrony). 
+The second confirmation rule defines the \emph{finalized chain}, and provides safety even under network partitions, but loses liveness 
+either under asynchrony or in case of fluctuation in the participation level.
+
+In this project, we targeted the 3SF specification in Python \footnote{URL to the Python specification:
+\url{https://github.com/freespek/ssf-mc/3sf.txt}} as the case study focussing only on the finality gadget protocol, which is mostly specified in the file \texttt{ffg.py}.
```
Comment (saltiniroberto) [inline reports/intro.tex:36]: Is the URL https://github.com/freespek/ssf-mc/3sf.txt correct?
That file does not exist at the moment.

Comment (konnov) [reply reports/intro.tex:36]: True. We could put a direct link to your commit hash, but it's a bit too long for a footnote. Any ideas?

File: reports/report.tex [hunk 3]
```diff
@@ -16,15 +16,16 @@
        Exploring Automatic Model-Checking of the Ethereum specification}
 \author{%
     Igor Konnov\thanks{This work was supported by Ethereum Foundation
-    under grant NNN.}\\ \small Independent Researcher \& TU Wien \\
+    under grant NNN.}\\ \small Independent Researcher \\
```
[the remark this answers is not in the document]
Comment (konnov) [reply reports/report.tex]: Just did, our grant numbers differ in the last digit :)

File: reports/intro.tex (hunk 2 above)
Comment (saltiniroberto) [reply reports/intro.tex:36]: I think that it is Ok to put the link to the commit hash. I tried it locally and the link is wrapper around, so it does not go over the margins.

File: reports/intro.tex (hunk 1 above)
Comment (banhday) [reply reports/intro.tex:22]: I have added a brief description for this

File: reports/intro.tex (hunk 2 above)
Comment (konnov) [reply reports/intro.tex]: I have added `\href{https://github.com/saltiniroberto/ssf/blob/ad3ba2c21bc1cd554a870a6e0e4d87040558e129/high_level/common/ffg.py}{https://github.com/saltiniroberto/ssf/.../ffg.py}`, so it's clickable and reasonably readable

**Comment by konnov** [comment]
I had a look at the report, and it looks good for me. Shall we merge it?

**Review by saltiniroberto — APPROVED** [review]

**Comment by saltiniroberto** [comment]
> I had a look at the report, and it looks good for me. Shall we merge it?

Yep. I have approved it. Leaving merging to you as I am not sure if you want to do Merge or Squash merge.

**Pull request merged**